### PR TITLE
add definition of junkie

### PR DESCRIPTION
**Pull request by calesennett**
- Adds definition of `junkie`
- Adds `junkie` as ableist language
- Updates vulnerable `npm` packages

### CODE REVIEW

**Comment by tatianamac** [comment]
Thank you for submitting, @calesennett and my apologies for taking so long to respond! I've put this out to the Self-Defined community on Slack to see if someone who has experienced the negative impact of this term could provide a review!

File: 11ty/definitions/junkie.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+alt_words:
+  - fanatic
+  - enthusiast
+  - lover
+  - fiend
+  - buff
+  - geek
+---
+
+a drug addict. a person with a compulsive habit or obsessive dependency on something.
```
Comment (kkemple) [inline 11ty/definitions/junkie.md]:
```suggestion
a drug addict. a person who suffers from a complex disease of the brain and body that involves compulsive use of one or more substances despite serious health and social consequences.
```

Comment (kkemple) [reply 11ty/definitions/junkie.md]: I've updated the definition of an addict to reflect the latest terminology that focuses on addiction as a disease.

File: 11ty/definitions/junkie.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Issues
+
+Junkie is commonly used as a term for someone whose enthusiasm or zeal for something is extreme or beyond normal limits. However, this diminishes the seriousness of drug addiction and substance use disorder.
```
Comment (kkemple) [inline 11ty/definitions/junkie.md]:
```suggestion
Junkie is commonly used as a term for someone whose enthusiasm or zeal for something may be considered extreme or beyond normal interest by others. However, this diminishes the seriousness of drug addiction and substance use disorder.
```

Comment (kkemple) [reply 11ty/definitions/junkie.md]: removing "is" as that states fact where this really falls under opinion

**Review by kkemple — CHANGES_REQUESTED** [review]
Thank you so much for the PR @calesennett! 💖

As someone who suffers from drug addiction and has been called a junkie, I've very happy to see this added to self defined. I've made some suggestions on the wording around the terminology for drug addict and some smaller suggestions around the issue definition. Would love to hear your thoughts!

Once again, thank you for submitting this PR!

**Comment by tatianamac** [comment]
@calesennett  Do you have bandwidth to address @kkemple 's suggestions? (No rush on our side; I just like to keep tasks moving.)

**Comment by calesennett** [comment]
@kkemple Thanks for suggesting better language around the definition and issue with using this word. I've updated the language as well as brought in the most recent `prod` branch. Thanks so much!

File: 11ty/definitions/junkie.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
+title: junkie
+slug: 'junkie'
```
Comment (tatianamac) [inline 11ty/definitions/junkie.md]:
```suggestion
slug: junkie
```

File: 11ty/definitions/junkie.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+speech: noun
+defined: true
+flag:
+  level: avoid
+  text: 'ableist language'
+alt_words:
```
Comment (tatianamac) [inline 11ty/definitions/junkie.md:9]:
```suggestion
excerpt: a drug addict. a person who suffers from a complex disease of the brain and body that involves compulsive use of one or more substances despite serious health and social consequences.
```

File: 11ty/definitions/junkie.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
+---
+title: junkie
+slug: 'junkie'
+speech: noun
+defined: true
+flag:
+  level: avoid
+  text: 'ableist language'
```
Comment (tatianamac) [inline 11ty/definitions/junkie.md:8]:
```suggestion
  text: 'ableist language'
  excerpt: a drug addict. a person who suffers from a complex disease of the brain and body that involves compulsive use of one or more substances despite serious health and social consequences.
```

File: 11ty/definitions/junkie.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+alt_words:
+  - fanatic
+  - enthusiast
+  - lover
+  - fiend
+  - buff
+  - geek
```
Comment (tatianamac) [inline 11ty/definitions/junkie.md]:
```suggestion
alt_words:
  - aficionado
  - buff
  - fanatic
  - enthusiast
  - fiend
  - geek
  - lover
```

File: 11ty/definitions/junkie.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+---
+
+a drug addict. a person who suffers from a complex disease of the brain and body that involves compulsive use of one or more substances despite serious health and social consequences.
```
Comment (tatianamac) [inline 11ty/definitions/junkie.md]:
```suggestion
a drug addict; a person who suffers from a complex disease of the brain and body that involves compulsive use of one or more substances despite serious health and social consequences
```

**Review by tatianamac — APPROVED** [review]
Added some semantic/syntax edits, which I wish I would have caught before. I tried to rely heavily on suggestions so you can add/reject seamlessly—hopefully it's not a burden! Thank you so much for submitting this definition!

**Pull request merged**